File: java/2011.java
class Solution {
    public int finalValueAfterOperations(String[] operations) {
        int x = 0;

        for(String a : operations){
            x += a.charAt(1) == '+' ? 1 : -1;
        }

        return x;
    }
}

class Main {
    public static void main(String[] args) {
        Solution solution = new Solution();

        String[] input1 = { "--X", "X++", "X++" };
        int result1 = solution.finalValueAfterOperations(input1);
        System.out.println(result1);
    }
}

File: kotlin/2011.kt
class Solution {
    fun finalValueAfterOperations(operations: Array<String>): Int {
        var returnValue = 0

        for(operation in operations){
            when (operation[1]) {
                '+' -> returnValue++
                else -> returnValue--
            }
        }

        return returnValue
    }
}

fun main() {
    val solution = Solution()

    val set1 = arrayOf("--X","X++","X++")
    val set2 = arrayOf("++X","++X","X++")

    println( solution.finalValueAfterOperations(set1))
    println( solution.finalValueAfterOperations(set2))
}